tests/dmctl_command/run.sh

```diff
@@ -86,6 +86,12 @@ function run() {
 	# query status with command mode
 	$PWD/bin/dmctl.test DEVEL --master-addr=:$MASTER_PORT query-status >$WORK_DIR/query-status.log
 
+	# use DM_MASTER_ADDR env
+	export DM_MASTER_ADDR="127.0.0.1:$MASTER_PORT"
```

Comment:
How about MASTER_ADDR ？

Reply:
now I think a DM prefix is better, because     it distinguishes itself from all environment variables.

Reply:
i think add DM as namespace is better?